sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -814,14 +912,28 @@ def _plot_ice_lines(
             n_ice_to_plot,
             replace=False,
         )
+        # copy, so that color list can be deleted and won't throw
+        # an error when passed to ax.plot()
+        individual_line_kw_copied = individual_line_kw.copy()
+        color_list = individual_line_kw.get("color_list", None)
+        individual_line_kw_copied.pop("color_list")
```

Comment:
just do `color_list = individual_line_kw_copied.pop("color_list")` and have both in one line